msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.ObjCBinding.CSharp.targets

```diff
@@ -46,10 +46,6 @@ Copyright (C) 2013-2016 Xamarin Inc. All rights reserved.
 		Inputs="$(MSBuildAllProjects);@(ObjcBindingApiDefinition);@(ObjcBindingCoreSource);@(ReferencePath);@(ObjcBindingNativeLibrary)"
 		Outputs="$(_GeneratedSourcesFileList)">
 
-		<ItemGroup>
-			<BTouchReferencePath Include="@(ReferenceCopyLocalPaths)" Condition="'%(Extension)' == '.dll'" />
-		</ItemGroup>
```

Comment:
This was added in maccore (before open sourcing the SDK):
```
commit f0a86283491f0267f55080e4ece49bcd65ebebc1
Author: Jeffrey Stedfast <jeff@xamarin.com>
Date:   Thu Oct 16 15:45:34 2014 -0400

    [msbuild] Attempt at fixing bug #22135
```

references:
https://bugzilla.xamarin.com/show_bug.cgi?id=22135
https://bugzilla.xamarin.com/show_bug.cgi?id=23674

so it's worth double checking the original test cases still works.